.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: "publish"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Set up Node"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
```

Comment:
I'm pretty sure we should bump it, but it's a fairly consequential change so I'll leave it to be done a bit latter